src/content/docs/CDN/CDN-case-study.md

```diff
@@ -0,0 +1,86 @@
+
+## Brief history
+
+In the 1990s a problem that was foreseen was that when the internet grew and more people tried to access the same websites at the same time there would be a bottleneck happening that would cause long waiting times and sometimes websites would crash, and this problem became a reality in the late 1990s. In the mid-1990s the first content delivery network was developed and the first commercial CDN was introduced in 1998 by Akamai as a more efficient way to deliver content on the Internet. In the early 2000s new CDN providers followed, and companies such as Limelight network, CacheFly, Cloudflare, and many others started to appear. Around 2003/2004 an increasing demand for audio and video started the next phase for CDNs and streaming was possible since CDNs offered a massively increased bandwidth, YouTube was released in 2005 and Netflix started streaming short time after in 2007 other streaming services followed soon after.
+
+This is a small bullet point list of some of the important milestones after 2007
+
+- **In 2008** Amazon launched a CDN integrated with their web service.
+- **In 2011** AT&T launched their new cloud-based CDN.
+- **In 2012** JSDelivr launched a free CDN for open-source project.
+- **In 2012** Netflix got its in-house CDN after an increase in streaming.
+- **In 2014** Apple got their in-house CDN.
+- **In 2015** BelugaCDN launched a new type of CDN price model a type of "pay as you go" driving costs lower.
+- **In 2023** Ai optimization, many big CDN providers has integrated Ai to predict traffic and implemented Ai real time decision making.
+
+## What is a Content delivery network?
+
+A content delivery network is a collection of servers around the world that are connected to a main server often called the origin server. The servers are strategically placed geographically across different countries and continents, and these servers are called POP (point of presence) servers or edge servers. The main function of these servers is to cache information such as HTML, JavaScript, CSS, and video content closer to the end user so loading time, performance, and reliability are improved, whereas a non-CDN network may take 5 seconds to load heavy website content to an end-user on the other side of the world a CDN network may have a pop server only a few miles from the end user that has cached heavy images and stylesheets so the loading time may be reduced to only 1 second.
+![Features](~/assets/CDN/OJ8XXP0.jpg)
+
+## Problem statement
+
+As talked about in the brief history section, a major problem with the state of networking in the 90s was slow speed, poor performance, and sometimes server crashes when there was too much traffic. This was a problem because of multiple factors such as
+
+- The Internet became more popular and grew rapidly and networking couldn't keep up.
+- People connected to websites that were hosted from a great geographical distance which caused long waiting times for the end-user.
```

Comment:
@mathiasg12 This is a good write-up. However, while I was reading, I felt like CDN is meant to solve the problem of the past. Can you establish the fact that CDN is a tool in this modern day? Can you demonstrate what the situation is like without/with it?

Reply:
thanks for the feedback I will add a section about a world with/without CDN

Reply:
You can use bootstrap to demonstrate

Reply:
I added some sentences at the end of the section "Main feature and objectives" explaining  a world without and with CDNs I also used bootstrap as an example of modern uses of CDNs